### PR TITLE
metric for blocks

### PR DESCRIPTION
**Pull request by tkstanczak**

### CODE REVIEW

File: src/Nethermind/Nethermind.Blockchain/Processing/BlockchainProcessor.cs [hunk 1]
```diff
@@ -281,7 +282,10 @@ public Block Process(Block suggestedBlock, ProcessingOptions options, IBlockTrac
 
             ProcessingBranch processingBranch = PrepareProcessingBranch(suggestedBlock, options);
             PrepareBlocksToProcess(suggestedBlock, options, processingBranch);
+
+            Stopwatch stopwatch = Stopwatch.StartNew();
             Block[] processedBlocks = ProcessBranch(processingBranch, options, tracer);
```
Comment (tkstanczak) [inline src/Nethermind/Nethermind.Blockchain/Processing/BlockchainProcessor.cs:287]: we want it to be the whole branch

File: src/Nethermind/Nethermind.Blockchain/Processing/BlockchainProcessor.cs [hunk 2]
```diff
@@ -281,7 +282,10 @@ public Block Process(Block suggestedBlock, ProcessingOptions options, IBlockTrac
 
             ProcessingBranch processingBranch = PrepareProcessingBranch(suggestedBlock, options);
             PrepareBlocksToProcess(suggestedBlock, options, processingBranch);
+
+            Stopwatch stopwatch = Stopwatch.StartNew();
```
Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.Blockchain/Processing/BlockchainProcessor.cs:286]: Why not reuse a stopwatch?

Comment (tkstanczak) [reply src/Nethermind/Nethermind.Blockchain/Processing/BlockchainProcessor.cs:286]: it is super light

Comment (tkstanczak) [reply src/Nethermind/Nethermind.Blockchain/Processing/BlockchainProcessor.cs:286]: oh, it is a class :)

Comment (tkstanczak) [reply src/Nethermind/Nethermind.Blockchain/Processing/BlockchainProcessor.cs:286]: cannot reuse it as we have multiple threads accessing this, would have to be threadlocal - better just create a light object which will be GCed after each block (not much)

**Pull request merged**